draft-ietf-sedate-datetime-extended.md

```diff
@@ -284,13 +284,12 @@ IANA Time Zone:
   <!-- The IANA time zone for `Z` is called `Etc/GMT`. Not true.  
   No idea which time zone name is preferred for Z. -->
 
-  Note that the rules defined for a named IANA time zone can change over
+  Rules defined for a named IANA time zone can change over
   time.
-  The use of a named IANA time zone implies that the intent is for the
-  rules that are current at the time of interpretation to apply, i.e.,
-  the additional information conveyed by using that time zone name is to
-  change with the changed rules as recorded in the IANA time zone
-  database.
+  Applications SHOULD use the most recent TZDB version to
```

Comment:
This incorrectly changes a statement of intent into a requirement on the recipient.